daemon/HTTPServer.cpp

```diff
@@ -1110,17 +1132,25 @@ namespace http {
 			}
 		}
 
-		// HTML head start
-		ShowPageHead (s);
-		if (req.uri.find("page=") != std::string::npos) {
-			HandlePage (req, res, s);
-		} else if (req.uri.find("cmd=") != std::string::npos) {
-			HandleCommand (req, res, s);
-		} else {
-			ShowStatus (s, true, i2p::http::OutputFormatEnum::forWebConsole);
-			res.add_header("Refresh", "10");
-		}
-		ShowPageTail (s);
+    if (req.uri == "/metrics") {
```

Comment:
I've decided to create a separate routing branch here since prometheus works with plain text only, `ShowPageHead` and `ShowPageTail` are not what we want